problem1/concrete/item_test.go

```diff
@@ -0,0 +1,170 @@
+package concrete
```

Comment:
use "github.com/stretchr/testify/require"

Reply:
use it like `require.Nil(t, err)`

Reply:
used require in one of the testcases